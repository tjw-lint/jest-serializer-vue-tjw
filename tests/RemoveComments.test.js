describe('RemoveComments', () => {
  test('Snapshot is unchanged', () => {
    const commented = `
<div>
  <span>3</span>
  <!---->
  <div>
    <!---->
    <span>2</span><!-- <div></div> -->
   <!--
     <div>
    </div>
   -->
    <div>
      <span>1</span>
      <!---->
      <!---->
    </div>
    <div>
      <div>
        <!--
        asdf asdf
       asdf
      -->
        <span>1</span>
        <!---->
        <!-- as as9-df df -->
        <!---->
        <!--asdf-->
        <!----><!---->
      </div>
    </div>
  </div>
  <div></div>
</div>
<!---->
`;

    expect(commented.trim())
      .toMatchSnapshot();
  });
});
