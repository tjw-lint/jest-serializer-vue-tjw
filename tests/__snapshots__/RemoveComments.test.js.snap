// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveComments Comments remain 1`] = `
<div>
  <span>3</span>
  <!---->
  <div>
    <!---->
    <span>2</span><!-- <div></div> -->
    <!--
     <div>
    </div>
   -->
    <div>
      <span>1</span>
      <!---->
      <!---->
    </div>
    <div>
      <div>
        <!--
        asdf asdf
       asdf
      -->
        <span>1</span>
        <!---->
        <!-- as as9-df df -->
        <!---->
        <!--asdf-->
        <!---->
        <!---->
      </div>
    </div>
  </div>
  <div></div>
</div>
<!---->
`;

exports[`RemoveComments Comments removed 1`] = `
<div>
  <span>3</span>
  <div>
    <span>2</span>
    <div>
      <span>1</span>
    </div>
    <div>
      <div>
        <span>1</span>
      </div>
    </div>
  </div>
  <div></div>
</div>
`;
